ci/before_deploy.sh

```diff
@@ -0,0 +1,26 @@
+set -ex
+
+main() {
+    local src=$(pwd) \
+          stage=
+
+    case $TRAVIS_OS_NAME in
+        linux)
+            stage=$(mktemp -d)
+            ;;
+    esac
+
+    test -f Cargo.lock || cargo generate-lockfile
+
+    cargo rustc --features $FEATURES --target $TARGET --bin $CRATE_NAME --release -- -C lto
+
+    cp target/$TARGET/release/$CRATE_NAME $stage/
+
+    cd $stage
+    tar czf $src/$CRATE_NAME-$TRAVIS_TAG-$TARGET.tar.gz *
```

Comment:
Why create a `.tar.gz` here instead of a `.zip`?

Reply:
It's more native for linux binaries, no?
I think it makes it easier to see which is a windows binary and which is a linux one, at least to non-tech savvy users (_What's a .tar.gz? I'll go with .zip_)

Reply:
We discussed over IRC and decided to go with `.tar.xz`.